GoMeet/src/main/webapp/test/spec/spec-interactive-map.js

```diff
@@ -93,6 +95,39 @@ describe('Post Location', function() {
 
     postLocation(TITLE_A, LAT_A, LNG_A, NOTE_A, mockedFetchWrapper);
 
-    expect(mockedFetchWrapper.doPost).toHaveBeenCalledWith('/location-data', expectedParams);
+    expect(mockedFetchWrapper.doPost).toHaveBeenCalledWith(
+        '/location-data', expectedParams);
   });
 });
+
+/** Test for fetch locations. */
+describe ('Fetch Locations', function() {
+  it ('Should create a marker for the location returned', async function() {
+    // Set up fake promise to return 
+    let promiseHelper;
+    let fetchPromise = new Promise(function(resolve, reject) {
+      promiseHelper = {
+        resolve: resolve,
+        reject: reject
+      };
+    });
+    const response = new Response(JSON.stringify([{title: "Hello",
+        lat: 10, lng: 15, note: "My Note"}]));
+    promiseHelper.resolve(response);
+
+    let mockedFetchWrapper = new FetchWrapper();
+    spyOn(mockedFetchWrapper, 'doGet').and.callFake(function() {
+      return fetchPromise});
+
+    // Set up mocks for Google MAPS API
+    const markerConstructorSpy = spyOn(google.maps, 'Marker');
+    const fakeMarker = jasmine.createSpyObj('Marker', ['addListener']);
+    markerConstructorSpy.and.returnValue(fakeMarker);
+    const infowindowConstructorSpy = spyOn(google.maps, 'InfoWindow');
+    let fakeMap = {};
+
+    await fetchLocations(fakeMap, mockedFetchWrapper);
+    expect(google.maps.Marker).toHaveBeenCalledWith(
+        {position: {lat: 10, lng: 15}, map: fakeMap});
```

Comment:
Should we also verify that the title and note were retrieved appropriately as well?